compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/intermediate/SafeExtensionInvokeCallReceiver.kt

```diff
@@ -49,7 +49,7 @@ class SafeExtensionInvokeCallReceiver(
             "Safe extension 'invoke' call should have null as its 1st value argument, got: ${callBuilder.irValueArgumentsByIndex[0]}"
         }
         callBuilder.irValueArgumentsByIndex[0] = safeReceiverValue.load()
-        val irResult = withDispatchAndExtensionReceivers(functionReceiver, null)
+        val irResult = withDispatchAndExtensionAndContextReceivers(functionReceiver, null, emptyList())
```

Comment:
```
fun test(ext: Any.() -> Any, b: Any?) =
    b?.ext()
```
Add an equivalent test for function-with-context-receiver type if possible.

Reply:
For now, we have a limitation on invoking contextual functional type. It is only possible by passing all arguments (including context receivers) explicitly like:

```kotlin
context(X)
fun f(g: context(X) (Int) -> Unit) {
    g(this@X, 42)
}
```

Reply:
I will describe the limitation in the KEEP.